grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractServerCall.java

```diff
@@ -540,7 +540,14 @@ static HttpHeaders statusToTrailers(
         GrpcTrailersUtil.addStatusMessageToTrailers(
                 trailersBuilder, status.getCode().value(), status.getDescription());
 
-        MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        try {
+            MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        } catch (Exception e) {
```

Comment:
Should we log `e`?

Reply:
Yeh, I think so too.
What do you think @jrhee17 ?

The other code like this has logged this kind of error
with log level of `WARN`.
It might be good to write the log to notify server implementer.

Reply:
Sure, sounds good